rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,59 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
+
+This feature is available for _Search Pattern_ and _Mouse Action_ Action Steps.
+
+When using this feature:
+
+* The Recording Toolbar is inactive.
+* Keyboard events are not recorded.
+
+=== Recommended Display Settings
+
+Apply the following displays settings to capture image-based mouse clicks:
+
+* Single monitor. Multiple monitors are supported, but the capturing will be limited to the primary display only for Bot tasks and to the web page for Web tasks.
+* Display scaling of 100%. If the scaling is different, then there is a risk that the captured image will not be found by the workflow generated in RPA Builder from the recording.
+
+To check your display settings before recording, go to *Windows Settings > Display*. If your *Scale and layout* or *Multiple displays* settings are different than what's recommended when recording, a warning message shows each time you record an image-based mouse click, and you will need to confirm the message each time it shows.
+
+=== Using Image-Based Mouse Clicks
+
+To capture image-based mouse clicks:
+
+. Start a normal recording process.
+. While recording, press *F3* to start recording image-based mouse clicks, or press the *Image-Based Mouse Clicks* button on the Recording Toolbar. This opens the wizard for capturing image-based mouse clicks.
+. Capture an image by selecting it on the screen, accept or retake it, and then set a mouse click. The mouse click is marked with a red dot.
+. Record additional mouse clicks, such as double-mouse clicks, if desired. All additional mouse clicks are recorded relative to the captured pattern and are marked with a red dot.
+. Click *Finish*. This generates all relevant data for RPA Builder to create the _Search Pattern_ and _Mouse Action_ Action Steps for the recording.
+
+To use image-based mouse clicks for the _Search Pattern_ Action Step:
+
+. Start a normal recording process.
+. While recording, press *F3* to start recording image-based mouse clicks, or press the *Image-based mouse clicks* button on the Recording Toolbar. This opens the wizard for the Action Step.
+. Select *Capture pattern*.
+. Draw a rectangular box to capture the desired pattern. The box shows a red frame around the capture. The search area is set to full-screen by default.
+. Select *Accept selection & continue* to continue with the relative mouse click action. All additional mouse clicks are recorded relative to the captured pattern and are marked with a red dot.
+. Click *Finish*. This generates all relevant data for RPA Builder to create the _Search Pattern_ Actions Steps for the recording.
+
+The data for the recording is included in the process documentation and the PDF generated by the Recorder. Once you have finished using the image-based mouse clicks feature, you can continue with the normal recording process.
+
+[NOTE]
+If you close the Action Step wizard by clicking the *X*, the image-based mouse clicks are not kept.
```

Comment:
Which dialog exactly is meant by "Action Step wizard"?
Action Steps are a concept from RPA Builder, there should not be any Action Step UI in the Recorder.